publication-rest/src/main/java/no/unit/nva/publication/delete/DeletePublicationHandler.java

```diff
@@ -127,6 +141,12 @@ private void handleSoftDeletion(RequestInfo requestInfo, Publication publication
         var duplicate = requestInfo.getQueryParameterOpt(DUPLICATE_QUERY_PARAM).orElse(null);
         resourceService.unpublishPublication(toPublicationWithDuplicate(duplicate, publication));
         persistNotification(publication);
+        eventBridgeClient.putEvents(PutEventsRequest.builder().entries(PutEventsRequestEntry.builder()
+               .source(NVA_PUBLICATION_DELETE_SOURCE)
+               .detailType(LAMBDA_DESTINATIONS_INVOCATION_RESULT_SUCCESS)
+               .detail(DoiMetadataUpdateEvent.createUpdateDoiEvent(publication).toJsonString())
+               .resources(publication.getIdentifier().toString())
+               .build()).build());
```

Comment:
Should be persisted if needed, etc. when publication has nva provided doi. Or not?

Reply:
I would extract this piece of code to new method:
`persistDoiUpdateEvent(publication);`